### PR TITLE
v0.6.2 kitchen sink

### PR DESCRIPTION
**Pull request by brimoor**
Change log
- Documenting the new `Dataset.version` property and its implications
- Renaming `fiftyone.core.labels._FrameLabel` to `_FrameLabels` because there are multiple "labels" attached to a frame
- Adding a `fiftyone.utils.data.load_video_labels()` utility for loading `eta.core.video.VideoLabels` into a frames dict
- Adds the Omission  `frames` from `get_field_schema()` for video datasets and samples
- Adds `Media type:` to view summaries

### CODE REVIEW

**Review by benjaminpkane — DISMISSED** [review]
LGTM!

File: docs/source/user_guide/using_datasets.rst [hunk 1]
```diff
@@ -138,6 +138,17 @@ Datasets are homogeneous; they must contain samples of the same media type:
     dataset.add_sample(fo.Sample(filepath="/path/to/video.mp4"))
     # MediaTypeError: Sample media type 'video' does not match dataset media type 'image'
 
+Dataset version
+------------------
+
+The version of the `fiftyone` package with which a dataset is created is stored
```
Comment (lethosor) [inline docs/source/user_guide/using_datasets.rst:144]: I might suggest "library" (also for other occurrences) for consistency, e.g. with index.rst

**Comment by brimoor** [comment]
@benjaminpkane If a migration happens, will the `version` be updated? Either up or down?

**Comment by brimoor** [comment]
Also, if it is updated, then this basically reflects the version of their package, or they'll never see this because the dataset failed to load. Conclusion is that perhaps we **don't** want to print this in `dataset.summary()`, since it is not actually informative.

File: docs/source/user_guide/using_datasets.rst (hunk 1 above)
Comment (brimoor) [reply docs/source/user_guide/using_datasets.rst:144]: I see package and library as interchangeable, except that I think we should use packgae when it is in specific reference to the `fiftyone` Python package or one of its attributes (like `version` here).

**Comment by benjaminpkane** [comment]
Yes, it can be hidden. The version is not meaningful to users.

File: docs/source/user_guide/using_datasets.rst (hunk 1 above)
Comment (lethosor) [reply docs/source/user_guide/using_datasets.rst:144]: I was considering this to refer to a more high-level "version of FiftyOne" than a low-level "`fiftyone.version`", but I guess either is fine with me.

**Review by benjaminpkane — APPROVED** [review]

**Pull request merged**